Feature/LP56 dashboard UI

Lupillo17 opened the pull request:
Finished: DashboardView
* view finished 
* added mock data only for test purpose 
* added string.xml file



Modified: LpGenericCard
* deleting modifier padding in the card
* accountNumber text padding modified and just apply at the end of the text
* accountNumber align text at the end of content
* accountNumber max lines = 1


UI example:
![image](https://user-images.githubusercontent.com/33376944/233411005-a8f2c61c-d9c3-4749-b590-b261f04596b5.png)

Lupillo17 committed:
Adding dashboard screen still in progress

Lupillo17 committed:
Still in progress.

Lupillo17 committed:
Merge from develop

Lupillo17 committed:
still in progress, blocked by components LPGenericElevatedCard and LpGenericCard

Lupillo17 committed:
Merge branch 'develop' into feature/LP56-dashboard-ui

Lupillo17 committed:
Merge branch 'develop' into feature/LP56-dashboard-ui

Lupillo17 committed:
Modified: LpGenericCard
* deleting modifier padding in the card
* accountNumber text padding modified and just apply at the end of the text
* accountNumber align text at the end of content
* accountNumber max lines = 1

Finished: DashboardView
* view finished
* added mock data only for test purpose
* added string.xml file

carlosph816 commented on feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt:
```diff
@@ -0,0 +1,122 @@
+import com.iteneum.designsystem.theme.LeasePertTheme
+import kotlin.random.Random
+
+@OptIn(ExperimentalMaterial3Api::class)
```
You need to put the class information please

Lupillo17 replied:
Comment solved

irv205 commented on feature/example/build.gradle:
```diff
@@ -40,4 +40,5 @@ dependencies {
 
     //Mudules
     implementation project(":core:designsystem")
+    implementation project(":feature:dashboard")
```
remove this implementation, only use this when you need to test your view, after that remove.

Lupillo17 replied:
solving

Lupillo17 replied:
Comment solved

irv205 commented on feature/dashboard/src/main/res/values/strings.xml:
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```
You can remove "dashboard", as each String file is individual it is not necessary to make more specifications

Lupillo17 replied:
Comment solved

Lupillo17 committed:
* Removed dashboard at the beginning of strings name
* Removed the dashboard implementation feature in build.gradle (example)
*Added class documentation

melissaquezada94 commented on feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt:
```diff
@@ -0,0 +1,130 @@
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .padding(sizes.regularSize)
```
Please add padding atributes

Lupillo17 replied:
@melissaquezada94 all the comments solved

melissaquezada94 commented on feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt:
```diff
@@ -0,0 +1,130 @@
+                showBadge = showBadgeNotification,
+                imageVector = Icons.Filled.Notifications
+            ) {
+                Log.e("tag", "go to notification")
```
Please add a `//TODO` comment to tell us which action is going to be executed in this button.

melissaquezada94 replied:
Please remove the all the Logs and the import log

Lupillo17 replied:
solved

melissaquezada94 commented on feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt:
```diff
@@ -0,0 +1,130 @@
+                showBadge = showBadgePerson,
+                imageVector = Icons.Filled.Person
+            ) {
+                Log.e("tag", "go to person")
```
Please add a `//TODO` comment to tell us which action is going to be executed in this button.

melissaquezada94 commented on feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt:
```diff
@@ -0,0 +1,130 @@
+            accountNumber = currentBalance,
+            currency = currentBalanceCurrency
+        ) {
+            Log.e("tag", "Go to payments")
```
Please add a //TODO comment to tell us which action is going to be executed in this button.

melissaquezada94 commented on feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt:
```diff
@@ -0,0 +1,130 @@
+            details = stringResource(R.string.view_detail),
+            accountNumber = serviceRequest,
+        ) {
+            Log.e("tag", "go to service request")
```
Please add a //TODO comment to tell us which action is going to be executed in this button.

melissaquezada94 commented on feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt:
```diff
@@ -0,0 +1,130 @@
+                Log.e("tag", "go to person")
+            }
+        }
+        Spacer(modifier = Modifier.size(sizes.minorRegularSize))
```
Instead use a spacer I recommend you use the padding between the cards

melissaquezada94 commented on feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt:
```diff
@@ -0,0 +1,130 @@
+        ) {
+            Log.e("tag", "Go to payments")
+        }
+        Spacer(modifier = Modifier.size(sizes.regularSize))
```
Instead use a spacer I recommend you use the padding between the cards

melissaquezada94 commented on feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt:
```diff
@@ -0,0 +1,130 @@
+        ) {
+            Log.e("tag", "go to service request")
+        }
+        Spacer(modifier = Modifier.size(sizes.regularSize))
```
Instead use a spacer I recommend you use the padding between the cards

melissaquezada94 commented on feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt:
```diff
@@ -0,0 +1,130 @@
+        ) {
+            Log.e("tag", "go to amenity reservations")
+        }
+        Spacer(modifier = Modifier.size(sizes.regularSize))
```
Instead use a spacer I recommend you use the padding between the cards

melissaquezada94 commented on feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt:
```diff
@@ -0,0 +1,130 @@
+            details = stringResource(R.string.view_detail),
+            accountNumber = amenityReservations
+        ) {
+            Log.e("tag", "go to amenity reservations")
```
Please add a //TODO comment to tell us which action is going to be executed in this button.

melissaquezada94 commented on feature/dashboard/src/main/java/com/iteneum/dashboard/presentation/DashboardView.kt:
```diff
@@ -0,0 +1,130 @@
+                    buttonText = "Go to Screen",
+                    color = getRandomColor(Random.nextInt(0, 3))
+                ) {
+                    Log.e("tag", event)
```
Please add a //TODO comment to tell us which action is going to be executed in this button.

melissaquezada94 replied:
Also I recommend you use a onClick high order function mock to sent in the event list

Lupillo17 committed:
Merge from develop

Lupillo17 committed:
Adding TODO's to specify the buttons actions

Adding padding attributes to the modifiers

Using padding inside of components instead of using Spacer component

Lupillo17 committed:
Remove Logs from code and its imports
